CHANGES.txt

```diff
@@ -1,6 +1,9 @@
 0.4 (unreleased)
 ----------------
 
+- Update the repo to standards.  See
+  https://github.com/Pylons/pyramid_chameleon/issues/22
+
```

Comment:
This may need merging, after today's commit dropping 3.3 support and adding 3.4, 3.5, 3.6, and 3.7 support.

Reply:
Rebased on master, merged differences, pushed, build succeeds. @tseaver could I get one more review from you, please? Thank you!

Reply:
I take it you mean to bump the next release to 1.0 in a separate PR after merging this one?

Reply:
Yup. 1.0 sounds good to me!